README.md

```diff
@@ -84,6 +84,14 @@ The Application ID, Search API Key, and Write API Key in Algolia's [API Keys set
 
 Similarly, you will use these values to set the `HERMES_WEB_ALGOLIA_APP_ID` and `HERMES_WEB_ALGOLIA_SEARCH_API_KEY` environment variables at build time.
 
+### Jira (optional)
+
+Jira can be optionally configured to enable linking Hermes projects with Jira issues.
+
+1. [Create an API token](https://support.atlassian.com/atlassian-account/docs/manage-api-tokens-for-your-atlassian-account/#Create-an-API-token) for Jira.
+
+1. Enable Jira using the `jira` block in the Hermes config file.
```

Comment:
Should we give an example of how to use the `jira` block in the config?

Reply:
Hmm, it's pretty well document directly in the example config: https://github.com/hashicorp-forge/hermes/blob/83ed46f97cb48fb61fdfe26f54dae5a2f9085247/configs/config.hcl#L174-L187 I think that should be enough?

Reply:
Yep, should be good!